### PR TITLE
Allow users to add a new item to the blacklist from config file

### PR DESCRIPTION
**Pull request by gort818**

### CODE REVIEW

File: src/vulkan.cpp [hunk 1]
```diff
@@ -2079,8 +2079,17 @@ static VkResult overlay_CreateInstance(
                              &instance_data->vtable);
    instance_data_map_physical_devices(instance_data, true);
 
+   parse_overlay_config(&instance_data->params, getenv("MANGOHUD_CONFIG"));
+
+   //check for blacklist item in the config file
+   std::stringstream ss(instance_data->params.blacklist);
+   std::string token;
+   while (std::getline(ss, token, ',')) {
```
Comment (jackun) [inline src/vulkan.cpp:2087]: Then probably make this `parse_str_tokenized` (or something, i can't name things :D ) parsed into a vector, for other APIs.

File: src/vulkan.cpp [hunk 2]
```diff
@@ -2079,8 +2079,17 @@ static VkResult overlay_CreateInstance(
...
+      trim(token);
+      add_blacklist(token);
```
Comment (jackun) [inline src/vulkan.cpp:2089]: Also vkCreateInstance can get called more then once so blacklist just gonna keep growing

File: src/vulkan.cpp (hunk 1 above)
Comment (gort818) [reply src/vulkan.cpp]: Ok now we have parse_str_tokenize .. I am not any better :)

File: src/vulkan.cpp (hunk 2 above)
Comment (gort818) [reply src/vulkan.cpp]: Hmm I am not seeing any duplicates in the blacklist, if that is what you meant.

Comment (gort818) [reply src/vulkan.cpp]: Ok implemented a check if item exits in blacklist

File: README.md [hunk 3]
```diff
@@ -188,7 +188,7 @@ Parameters that are enabled by default have to be explicitly disabled. These (cu
 | `cellpadding_y`                    | Set the vertical cellpadding, default is `-0.085` |
 | `frametime`                        | Display frametime next to fps text                                                    |
 | `table_columns`                    | Set the number of table columns for ImGui, defaults to 3                              |
-| `blacklist`                        | Add a program to the blacklist. e.g `blacklist=vkcube,WatchDogs2.exe`                 |
+| `blacklist`                        | Add a program to the blacklist. e.g `blacklist=vkcube + WatchDogs2.exe`                 |
```
Comment (flightlessmango) [inline README.md:191]: shouldn't be spaces between the blacklist names and the `+`

Comment (gort818) [reply README.md:191]: Because of aesthetics? or functionality?  `parse_str_tokenize` is calling trim() on the string

Comment (flightlessmango) [reply README.md:191]: I'm assuming it would not work well with the env var

Comment (gort818) [reply README.md:191]: Ahh ok yep makes sense

**Pull request merged**